css/layout.css

```diff
@@ -32,3 +34,13 @@
     width: 50%;
   }
 }
+
+#entries-title {
+  display: inline;
+}
```

Comment:
This is not necessary, since it is already in a `flex` container. Remember, the default behavior of a flex container is to make all children sit on one line